Sources/GRPC/GRPCServerPipelineConfigurator.swift

```diff
@@ -78,38 +78,39 @@ final class GRPCServerPipelineConfigurator: ChannelInboundHandler, RemovableChan
   }
 
   /// Makes an HTTP/2 handler.
-  private func makeHTTP2Handler() -> NIOHTTP2Handler {
-    return .init(
-      mode: .server,
-      initialSettings: [
-        HTTP2Setting(
-          parameter: .maxConcurrentStreams,
-          value: self.configuration.httpMaxConcurrentStreams
-        ),
-        HTTP2Setting(
-          parameter: .maxHeaderListSize,
-          value: HPACKDecoder.defaultMaxHeaderListSize
-        ),
-        HTTP2Setting(
-          parameter: .maxFrameSize,
-          value: self.configuration.httpMaxFrameSize
-        ),
-        HTTP2Setting(
-          parameter: .initialWindowSize,
-          value: self.configuration.httpTargetWindowSize
-        ),
-      ]
-    )
-  }
-
-  /// Makes an HTTP/2 multiplexer suitable handling gRPC requests.
-  private func makeHTTP2Multiplexer(for channel: Channel) -> HTTP2StreamMultiplexer {
-    var logger = self.configuration.logger
+  private func makeHTTP2Handler(
+    for channel: Channel,
+    streamDelegate: NIOHTTP2StreamDelegate? = nil
```

Comment:
For internal/private functions I tend to avoid defaults, it avoids forgetting to pass in values.
 
```suggestion
    streamDelegate: NIOHTTP2StreamDelegate?
```